packages/block-library/src/post-title/edit.js

```diff
@@ -114,20 +119,22 @@ export default function PostTitleEdit( {
 
 	return (
 		<>
-			<BlockControls group="block">
-				<HeadingLevelDropdown
-					selectedLevel={ level }
-					onChange={ ( newLevel ) =>
-						setAttributes( { level: newLevel } )
-					}
-				/>
-				<AlignmentControl
-					value={ textAlign }
-					onChange={ ( nextAlign ) => {
-						setAttributes( { textAlign: nextAlign } );
-					} }
-				/>
-			</BlockControls>
+			{ blockEditingMode === 'default' && (
+				<BlockControls group="block">
+					<HeadingLevelDropdown
+						selectedLevel={ level }
+						onChange={ ( newLevel ) =>
+							setAttributes( { level: newLevel } )
+						}
+					/>
+					<AlignmentControl
+						value={ textAlign }
+						onChange={ ( nextAlign ) => {
+							setAttributes( { textAlign: nextAlign } );
+						} }
+					/>
+				</BlockControls>
+			) }
```

Comment:
I'm still able to edit/interact with block toolbars in the header when in content editing mode, though the toolbars no longer show the block type:

 
<img width="273" alt="Screenshot 2023-05-25 at 1 27 28 pm" src="https://github.com/WordPress/gutenberg/assets/8096000/a5d0a0c0-a3a9-4184-ba4e-3d454a462751">

<img width="275" alt="Screenshot 2023-05-25 at 1 27 57 pm" src="https://github.com/WordPress/gutenberg/assets/8096000/966b8e4a-e156-40d4-ad37-a49467fb764f">

Reply:
Huh, I don't see that... what steps did you take?

Reply:
I can too - site title and paragraphs in the header and footer seem to be editable. Not sure I did anything special.

I also noticed that when switching from editing the template back to editing the page content, the block you have selected might be part of the template, but it remains selected and has a toolbar.

Reply:
I noticed the same. This PR is otherwise testing very nicely for me, though!

Here's a screengrab in case it helps:

https://github.com/WordPress/gutenberg/assets/14988353/d7e22204-454b-40d6-8d3a-9d42260936ba

Reply:
Do you have the latest changes pulled? There was a bug where you could select a block by double clicking or dragging that should be fixed now.

Reply:
> Do you have the latest changes pulled?

Yes, just tried re-pulling, and my environment says it's up to date, and I re-ran `npm run dev`. In the screengrab, I'm single-clicking the Site Title block in the header, but I can also reproduce by clicking the Site Title or Paragraph blocks in the footer, too.

Reply:
I have latest changes too.

Reply:
Are you using TT3?

Reply:
Yep!

Reply:
Aha, looks like it only happens in Chrome. I'll investigate tomorrow.